docs/ruleset_functions/is_zone_mechanically_cooled.md

```diff
@@ -0,0 +1,30 @@
+# is_zone_mechanically_cooled
+**Schema Version:** 0.0.22  
+
+**Description:** determines whether a zone is mechanically cooled or not.  DOES NOT CHECK FOR TRANSFER AIR
```

Comment:
I don't think transfer air needs to be considered here, so I am OK with it as is.